### PR TITLE
Set up monorepo and add templates

### PR DESCRIPTION
**Pull request by remidej**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,8154 @@
...
+  integrity sha1-wc56MWjIxmFAM6S194d/OyJfnDg=
+
+html-tags@^3.1.0:
+  version "3.1.0"
+  resolved "https://registry.yarnpkg.com/html-tags/-/html-tags-3.1.0.tgz#7b5e6f7e665e9fb41f30007ed9e0d41e97fb2140"
+  integrity sha512-1qYz89hW3lFDEazhjW0yVAV87lw8lVkrJocr72XmBkMKsoSVJCQx3W8BXsC7hO2qAt8BoVjYjtAcZ9perqGnNg==
+
+html-to-react@^1.3.4:
+  version "1.4.7"
+  resolved "http://localhost:4873/html-to-react/-/html-to-react-1.4.7.tgz#a58129c1b77c6d4e047a647372bd194e25420b89"
```
Comment (markkaylor) [inline yarn.lock:3585]: 🤔 how come there are so many packages set to localhost?

Comment (remidej) [reply yarn.lock:3585]: Looks like some of them are pointing to Verdaccio, I re-build the yarn lock file. Thanks

File: packages/starters/next-blog/starter/lib/media.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { getStrapiURL } from "./api"
+
+export function getStrapiMedia(media) {
+  console.log("GET MEDIA", media)
```
Comment (alexandrebodin) [inline packages/starters/next-blog/starter/lib/media.js:4]: is it necessary ?

File: packages/starters/next-blog/starter/pages/_app.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import { getStrapiMedia } from "../lib/media"
+
+// Store Strapi Global object in context
+export const GlobalContext = createContext({})
+
+const MyApp = ({ Component, pageProps }) => {
+  const { global } = pageProps
+
+  // console.log("GLOBAL", global)
+  console.log("GLOBAL get image myApp")
```
Comment (alexandrebodin) [inline packages/starters/next-blog/starter/pages/_app.js:15]: same question here

File: packages/starters/next-corporate/starter/components/layout.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { useState } from "react"
+import Navbar from "./elements/navbar"
+import Footer from "./elements/footer"
+import NotificationBanner from "./elements/notification-banner"
+
+const Layout = ({ children, global, pageContext }) => {
+  console.log("GLOBAL", global)
```
Comment (alexandrebodin) [inline packages/starters/next-corporate/starter/components/layout.js:7]: logggggg

**Review by alexandrebodin — COMMENTED** [review]
That's impossible to review all at once to be honest. As it's only about migrating we will just have to make sure they all work fine in beta.

**Pull request merged**